client/app/telecom/telephony/alias/configuration/feature/conference/telecom-telephony-alias-configuration-conference.html

```diff
@@ -125,44 +125,50 @@
                data-ng-model="$ctrl.conference.reportEmail"
                placeholder="{{:: 'telephony_alias_config_conference_custom_email_placeholder' | translate }}"
                required>
+    </form>
 
-        <label class="oui-label"
-               data-translate="telephony_alias_config_conference_followup_control_conference"></label>
-        <oui-spinner data-ng-if="$ctrl.generatingUrls"></oui-spinner>
-
-        <div data-ng-if="!$ctrl.generatingUrls">
-            <button class="oui-button oui-button_secondary oui-button_icon-left mb-5"
-                    data-ng-if="!$ctrl.hasWebAccessUrls"
-                    data-ng-click="$ctrl.generateWebAccess()">
-                <span class="oui-icon oui-icon-add"
-                      aria-hidden="true"></span>
-                <span data-translate="telephony_alias_config_conference_generate_urls"></span>
-            </button>
+    <label class="oui-label"
```

Comment:
`label` linked to nothing (where is the `for`)

Reply:
No element to link to => going to be a `<span>`